.gitignore

```diff
@@ -79,6 +79,3 @@ flattened.sol
 
 # Environment
 .env
-
-# Directories
-/generated/
```

Comment:
The interface generation script does not populate `/generate/` anymore but directly writes in the correct output folders. Unrelated, could move it to a seperate PR.